### ISSUE TITLE
Filter allowing theme/plugin devs to put classes on pym shortcode output

### ISSUE THREAD

**Issue by benlk** [INN/pym-shortcode#22]
For example, if it was desired that all pym embed parent divs should have the class `pym_parent`:

```php
/**
 * add 'pym_parent' to the classes on the pym field array
 * @param Array $classes
 * @return Array
 */
function my_pym_class_filter( $classes ) {
    $classes[] = 'pym_parent';
    return $classes;
}
add_filter( 'pym_element_classes', 'my_pym_class_filter' );
```

**Comment by rclachance** [comment]
what's the use case for this?

**Comment by benlk** [comment]
Right now there's no easy way to apply CSS styles specific to pym embeds. There are no selectors specific to the div containing the iframe or the iframe itself. The markup looks like this:

```html
<div id="pym_0">
    <iframe src="http://f.benlk.com/graphics/heartbeat-abortion-bills/child.html?initialWidth=940&amp;childId=pym_0&amp;parentTitle=Settling%20for%20misconduct%20%7C%20Chicago%20Reporter&amp;parentUrl=http%3A%2F%2Fchicagoreporter.vagrant.dev%2Fseries%2Fsettling-for-misconduct%2F" width="100%" scrolling="no" marginheight="0" frameborder="0" height="1123px"></iframe>
</div>
<script src="http://chicagoreporter.vagrant.dev/wp-content/plugins/pym-shortcode/js/pym.v1.min.js"></script>
<script>
    var pym_0 = new pym.Parent('pym_0', 'http://f.benlk.com/graphics/heartbeat-abortion-bills/child.html', {})
</script>
```

If someone wanted to style an embed, say to make it wider than its container or something, they'd need to either put the shortcode inside another element or they'd need to target the `#pym_0` ID, which isn't guaranteed. This is kind of a parallel to #21's request to allow people to set the element ID of the pym embed, but in a general sense. Someone could write styles like this:

```less
#pym_0,
#pym_1,
#pym_2 {
    max-width: 400px; // I don't know
}
```

But that's kinda hacky, and causes code duplication if they already have a predefined class that would have the same effect.

This isn't needed for anything at the moment, so I marked it `priority:low`

**Comment by rclachance** [comment]
yeah, I'd say just add a `pym` class to the default rendering and then we can implement #21

**Comment by benlk** [comment]
Closed with #27.
